changelog/change_update_style_redundant_format_to_register_offenses_20250321110550.md

```diff
@@ -0,0 +1 @@
+* [#14015](https://github.com/rubocop/rubocop/pull/14015): Update `Style/RedundantFormat` to register offenses when the only argument to `format` or `sprintf` is a constant. ([@dvandersluis][])
```

Comment:
```suggestion
* [#14016](https://github.com/rubocop/rubocop/pull/14016): Update `Style/RedundantFormat` to register offenses when the only argument to `format` or `sprintf` is a constant. ([@dvandersluis][])
```

Reply:
Oops, I even saw you had opened 14015 before I pushed but I typed the wrong thing 😅 

Fixed now.